backend/sample.env

```diff
@@ -120,8 +120,8 @@ SESSION_COOKIE_AGE=86400
 # Time in seconds to wait before polling LLMWhisperer's status API
 ADAPTER_LLMW_POLL_INTERVAL=30
 # Total number of times to poll the status API.
-# 500 mins to allow 1500 (max pages limit) * 20 (approx time in sec to process a page)
-ADAPTER_LLMW_MAX_POLLS=1000
+# ~60 mins (assuming it'll be enough to process 1500 pages with LLMW v2)
+ADAPTER_LLMW_MAX_POLLS=120
```

Comment:
@chandrasekharan-zipstack Will this not cause timeout if it exceeds max 15min time which we have set in gunicorn ?

Reply:
@harini-venkataraman the timeout exists only for extractions that happens with a web UI. This large setting is mainly ideal for async pipeline based extractions where such gunicorn timeouts will not play a role